api/context.go

```diff
@@ -431,10 +432,22 @@ func IsPrivateIpAddress(ipAddress string) bool {
 }
 
 func RenderWebError(err *model.AppError, w http.ResponseWriter, r *http.Request) {
+
+	protocol := "http"
+	if utils.Cfg.ServiceSettings.UseSSL {
```

Comment:
not sure the UseSSL check is needed, but it's ok.  We need to cleanup UseSSL, but that's out of scope for this tix.